package.json

```diff
@@ -1,6 +1,37 @@
 {
     "name": "vscode-cfn-lint",
-    "description": "CloudFormation linter",
+    "displayName": "CloudFormation Linter",
+    "description": "AWS CloudFormation Linter",
+    "icon": "logo.png",
+    "keywords": [
```

Comment:
It says [here](https://code.visualstudio.com/api/references/extension-manifest#fields) that you shouldn't put more than 5 keywords

Reply:
`"This list is currently limited to 5 keywords."` doesn't seem accurate:

https://github.com/dsteenman/cloudformation-yaml-snippets/blob/aeca3a451b2ae542decffc87fbacba5c0a973756/package.json#L28-L42

https://marketplace.visualstudio.com/items?itemName=dsteenman.cloudformation-yaml-snippets

Reply:
Hmm then maybe put the most important 5 at the top just in case 🤷